docs/FAQs.md

```diff
@@ -0,0 +1,37 @@
+### Does Karpenter replace the Kube Scheduler?
+No. Provisioners work in tandem with the Kube Scheduler. When capacity is unconstrained, the Kube Scheduler will schedule pods as usual. It may schedule pods to nodes managed by Provisioner or other types of capacity in the cluster. Provisioners only attempt to schedule pods of when `type=PodScheduled,reason=Unschedulable`. In this case, they will make a provisioning decision, launch new capacity, and bind pods to the provisioned nodes. Provisioners do not wait for the Kube Scheduler to make a scheduling decision in this case, as the decision is already made by nature of making a provisioning decision.
+### Does Karpenter support node selectors?
+Yes. Node selectors are an opt-in mechanism which allows customers to specify the nodes to which a pod can schedule. Provisioners recognize well known node selectors on incoming pods and use them to constrain the nodes they generate. For example, well known selectors like `node.kubernetes.io/instance-type`, `topology.kubernetes.io/zone`, `kubernetes.io/os`, `kubernetes.io/arch` are supported, and will ensure that provisioned nodes are constrained accordingly. Additionally, customers may specify arbitrary labels, which will be automatically applied to every node launched by the Provisioner.
+### Does Karpenter support taints?
+Yes. Taints are an opt-out mechanism which allows customers to specify the nodes to which a pod cannot schedule. Unlike labels, Provisioners do not automatically taint nodes in response to pod tolerations, since pod tolerations do not require that corresponding taints exist. However, similar to labels, customers may specify taints for their Provisioner, which will automatically be applied to every node in the group. This means that if a Provisioner is configured with taints, any incoming pods will not be provisioned unless they tolerate the taints.
+### Does Karpenter support topology spread constraints?
+Yes. Provisioners respect `pod.spec.topologySpreadConstraints`. Allocating pods with these constraints may yield highly fragmented nodes, due to their strict nature and complexity of “online binpacking” algorithms. However, the reallocation pass is able to produce much more efficient packings using “offline binpacking” techniques.
+### Does Karpenter support affinity?
+No. Karpenter intentionally does not support affinity due the to scalability limitations outlined by SIG Scalability. Do you have a use case for affinity? We're excited to hear about it in our [Working Group](working-group/README.md).
+### Does Karpenter support custom resource like accelerators or HPC?
+Yes. Support for specific custom resources can be implemented by your cloud provider.
+### Does Karpenter support daemonsets?
```

Comment:
DaemonSets?

Reply:
I ended up using the lowercase kubectl syntax (nodes, daemonsets) instead of the UpperCamelProperNouns, since too much of the doc was feeling like proper nouns.